help/destinations/catalog/crm/zendesk.md

```diff
@@ -175,9 +186,11 @@ This section captures the functionality and significant documentation updates ma
 
 |Release month|Update type|Description|
 |---|---|---|
-|April 2023|Documentation update| <ul><li>We updated the [use-cases](#use-cases) section with a clearer example of when customers would benefit from using this destination.</li> <li>We updated the [mapping](#mapping-considerations-example) section to reflect the correct required mappings. The `Attribute: last_name` and `Identity: email` target mappings are mandatory for this destination. If these mappings are missing, any other mappings are ignored and not sent to [!DNL Zendesk].</li> <li>We updated the [mapping](#mapping-considerations-example) section with clear examples of both mandatory and optional mappings.</li></ul> |
+|November 2023|Functionality update|<ul><li>You can now activate external audiences originating from custom uploads to this destination. See the [supported audiences](#supported-audiences) section for more details.</li><li>The **[UICONTROL Mapping ID]** values in the [schedule audience export](#schedule-segment-export-example) step are now automatically converted to lowercase values. Additionally, spaces and special characters are now automatically sanitized to ensure compliance with [!DNL Zendesk] field names.</li></ul>|
+|April 2023|Documentation update| <ul><li>The [use-cases](#use-cases) section was updated, You can now find a clearer example of when customers would benefit from using this destination.</li> <li>The [mapping](#mapping-considerations-example) section was updated to reflect the correct required mappings. The `Attribute: last_name` and `Identity: email` target mappings are mandatory for this destination. If these mappings are missing, any other mappings are ignored and not sent to [!DNL Zendesk]. Additionally, added clear examples of both mandatory and optional mappings.</li></ul> |
```

Comment:
On the April entry, please add the last sentence as a different bullet point.

- Added clear examples of both mandatory and optional mappings.